README.md

```diff
@@ -29,6 +29,13 @@ Stacktrace:
    @ REPL[3]:1
 ```
 
+Note regarding the return type of `@spawn`:
+* `Threads.@spawn` returns a value of the concrete type `Task`
+* In contrast, the return type of `StableTasks.@spawn` may depend on any and all of the following:
+    * the configuration of type inference in the Julia compiler
+    * the return type of the input function
+        * in particular, `typeof(StableTasks.@spawn 3) != typeof(StableTasks.@spawn 3.0)`, because `typeof(3) !== typeof(3.0)`
+
```

Comment:
```suggestion
* You should **not** write code that relies upon the exact return type of `StableTasks.@spawn` being concrete, or stable across versions. The return type in its parameter is available **only** as an optimization.

```
Maybe just to strengthen this a bit and make it clear that users shouldn't rely on this type.